docs/standard-library/versioning/reference/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Index
+sidebar_position: 0
+toc_min_heading_level: 2
+toc_max_heading_level: 3
+---
```

Comment:
@allenjzhang I don't understand why this is generating an empty index.md. The versioning library definitely contains decorators and so forth.

Reply:
Nevermind. It was because the `regen-ref-docs.mjs` script had `Versioning` instead of `TypeSpec.Versioning`.